src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -1367,4 +1367,39 @@ public void testDataSourceOktaGenerates429StatusCode() throws Exception {
       thread.join();
     }
   }
+
+  @Test
```

Comment:
let's add the javadoc above the test methods in which version we are going to add the feature e.g. `added in > 3.16.1`